src/Compilers/CSharp/Portable/FlowAnalysis/ControlFlowPass.cs

```diff
@@ -350,7 +353,9 @@ public override BoundNode VisitGotoStatement(BoundGotoStatement node)
                 }
                 else if (sourceStart > usingStart && targetStart < usingStart)
                 {
-                    // Backwards jump, so we must have already seen the label
+                    // Backwards jump, so we must have already seen the label, or it must be a switch case label. If it is a switch case label, we know
```

Comment:
Is this comment still correct? We won't see the case labels now right, because we `continue` above?

Reply:
No, we'll still see it here. We forward-define all switch case labels.